projects/project-player-library/src/lib/shared/attachment-show-card/attachment-show-card.component.html

```diff
@@ -25,9 +27,11 @@
                 {{attachments.name}}
             </div>
         </div>
-        <div class="flex flex-col cursor-pointer" (click)="actionEmit(attachments)">
-            <div class="flex justify-center items-end"><mat-icon fontIcon="cancel" class="primary-icon"></mat-icon></div>
-            <div class="text-xs primary-icon">{{ "REMOVE" | translate}}</div>
-        </div>
+        @if ( projectDataStatus !== 'submitted' ) {
```

Comment:
Use the 'submitted' value from the constant file

Reply:
done